src/main/java/com/aws/greengrass/clientdevices/auth/iot/registry/RegistryConfig.java

```diff
@@ -6,5 +6,7 @@
 package com.aws.greengrass.clientdevices.auth.iot.registry;
 
 public class RegistryConfig {
-    public static final int REGISTRY_CACHE_SIZE = 50;
+    protected static final int REGISTRY_CACHE_SIZE = 50;
+    protected static final int REGISTRY_CACHE_ENTRY_TTL_SECONDS = 24 * 60 * 60; // 1 day
+    protected static final int REGISTRY_REFRESH_FREQUENCY_SECONDS = 24 * 60 * 60; // 1 day
```

Comment:
Do you think this is the config we want to expose? What if there is no network when the refresh cycle runs? How will you handle intermittent cloud failures?